### ISSUE TITLE
warning NU1701: Package 'EntityFrameworkCore.Jet 2.2.0' was restored using

### ISSUE THREAD

**Issue by dmitrybv** [bubibubi/EntityFrameworkCore.Jet#66]
Hello
When I try to use EntityFrameworkCore.Jet in net5.0 application it gives next warning.

Users.csproj : warning NU1701: Package 'EntityFrameworkCore.Jet 2.2.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0-windows7.0'. This package may not be fully compatible with your project.

Is that how it should be?

**Comment by lauxjpn** [comment]
You will need to use the 3.x compatible version of EFCore.Jet, which hasn't been officially released yet. But you can just compile it yourself by using the [3.1-preview](https://github.com/bubibubi/EntityFrameworkCore.Jet/tree/3.1-preview) branch.

I will try to put some time aside this week to at least get i prerelease version into a nightly build feed.

**Comment by dmitrybv** [comment]
Ok. Thanks.

**Comment by ChrisJollyAU** [comment]
@dmitrybv Any success with using the 3.1-preview branch on net5.0? I did give it a go this weekend and while the main project that was using it compiled I did get some runtime errors (missing references and missing constructors). Admittedly other parts of the project were using ef core 5 while that branch uses ef core 3.1 which obviously doesn't agree. Maybe a combination of net5 and ef core 3 might work.

@lauxjpn I do have an experimental branch for net5 that does work. Has to be compiled for net5.0-windows7.0 . Have confirmed querying tables does work. Not sure on the mileage of the tests as some work and some don't. EFCore.Jet.FunctionalTests doesnt even compile (around 1500 errors)
